cmake/pcl_all_in_one_installer.cmake

```diff
@@ -22,61 +22,63 @@ if(BUILD_all_in_one_installer)
         list(APPEND PCL_3RDPARTY_COMPONENTS ${dep})
     endforeach(dep)
 
-    if(CMAKE_CL_64)
-        set(OPENNI_PACKAGE "OpenNI-Win64-1.5.4-Dev.msi")
-        set(OPENNI_URL "http://sourceforge.net/projects/pointclouds/files/dependencies/${OPENNI_PACKAGE}")
-        set(OPENNI_MD5 c8f9cbe8447a16d32572a4e2c2d00af0)
-        set(OPENNI_SENSOR_PACKAGE "Sensor-Win-OpenSource64-5.1.0.msi")
-        set(OPENNI_SENSOR_URL "http://sourceforge.net/projects/pointclouds/files/dependencies/${OPENNI_SENSOR_PACKAGE}")
-        set(OPENNI_SENSOR_MD5 badb880116436870943b1b7c447dfa22)
-    else(CMAKE_CL_64)
-        set(OPENNI_PACKAGE "OpenNI-Win32-1.5.4-Dev.msi")
-        set(OPENNI_URL "http://sourceforge.net/projects/pointclouds/files/dependencies/${OPENNI_PACKAGE}")
-        set(OPENNI_MD5 996d48f447b41a5501b7d22af27ab251)
-        set(OPENNI_SENSOR_PACKAGE "Sensor-Win-OpenSource32-5.1.0.msi")
-        set(OPENNI_SENSOR_URL "http://sourceforge.net/projects/pointclouds/files/dependencies/${OPENNI_SENSOR_PACKAGE}")
-        set(OPENNI_SENSOR_MD5 55da1f7541d7c9c98772bddf801c7e1c)	
-    endif(CMAKE_CL_64)
+    if(WITH_OPENNI)
```

Comment:
Only include installer of OpenNI/Sensor to the All-in-one Installer when WITH_OPENNI is selected in CMake.